code/datums/shuttles.dm

```diff
@@ -299,20 +304,23 @@
 	extra_desc = "This shuttle costs 500 credits to board."
 	admin_notes = "Due to the limited space for non paying crew, this shuttle may cause a riot."
 	credit_cost = 10000
+	danger_level = SHUTTLE_DANGER_SUBPAR
 
 /datum/map_template/shuttle/emergency/discoinferno
 	suffix = "discoinferno"
 	name = "Disco Inferno"
 	description = "The glorious results of centuries of plasma research done by Nanotrasen employees. This is the reason why you are here. Get on and dance like you're on fire, burn baby burn!"
 	admin_notes = "Flaming hot. The main area has a dance machine as well as plasma floor tiles that will be ignited by players every single time."
 	credit_cost = 10000
+	danger_level = SHUTTLE_DANGER_HIGH
 
 /datum/map_template/shuttle/emergency/arena
 	suffix = "arena"
 	name = "The Arena"
 	description = "The crew must pass through an otherworldy arena to board this shuttle. Expect massive casualties. The source of the Bloody Signal must be tracked down and eliminated to unlock this shuttle."
-	admin_notes = "RIP AND TEAR."
+	admin_notes = "RIP AND TEAR. Creates an entire internal Z-level where you have to kill each other in a massive battle royale to get to the actual shuttle."
```

Comment:
please replace "battle royale"

Reply:
Don't confuse react me it's not battle royale and I'm sick of hearing this @Tyranicranger4

Reply:
Then what IS it

Reply:
Certainly not battle royale
Especially since we have actual battle royale gamemode

Reply:
I've seen this shuttle ingame twice. The one time I remember I got murdered in a massive chainsaw fight to the death. How is that not a battle royale

Reply:
Just change it to a synonim like deathmatch or Free for all

Reply:
it completely matches the definition of a battle royale. also no one sees the description except admins. It very accurately describes what occurs (a battle royale)

Reply:
I don't think it matters much either, it's an admin only thing anyhow